rules/eslint.js

```diff
@@ -7,6 +7,12 @@ module.exports = {
     'consistent-return': 'off',
     // Ignored properties in TypeScript consist of underscores.
     'id-length': ['error', { exceptions: ['_', '__', '___', '____'] }],
+    // Using functions and firestore, and Prettier formatting result in extra
+    // lines being used.
+    'max-lines-per-function': ['error', { max: 70 }],
```

Comment:
setting it as `error` this would require a lot of changes on one go
I would mark them as warnings for now and be more strict during the PR process